src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,27 @@
+.forecast-summary {
+  text-align: center;
+}
+
+.forecast-summary .forecast-summary__icon i {
```

Comment:
Rather than using nested selectors like this, you could just add another class to the i element and target that directly to reduce the specificity

Reply:
Oh that's a good shout, I don't know why I didn't try to add a class to the WeatherIcon component, I guess I didn't think I could pass className as a prop to it